doc/cug.tex

```diff
@@ -3558,6 +3558,50 @@ \subsubsection{Trigger Types}
                 """
 \end{lstlisting}
 
+\paragraph{Writing Efficient Inter-Family Triggering}
+\label{EfficientInterFamilyTriggering}
+
+While cylc allows writing dependencies between two families such as this:
+
+\begin{lstlisting}
+[scheduling]
+    [[dependencies]]
+        graph = """
+    FAM1:start-any => FAM2
+                """
+\end{lstlisting}
+
+it is important to consider the number of dependencies this will generate. In
+the above example, each member of \lstinline=FAM2= has dependencies pointing at
+all the members of \lstinline=FAM1=. Expanding this out, you generate
+\lstinline=N * M= dependencies, where \lstinline=N= is the number of members of
+\lstinline=FAM1= and \lstinline=M= is the number of members of
+\lstinline=FAM2=. As a result, the suite can have an increasingly large memory
+footprint as the number of members of these families grows.
```

Comment:
Instead of *As a result, the suite can have an increasingly large memory footprint as the number of members of these families grows*, can we just say something like *For large families this can have a large memory footprint.*